<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZExceptionDLg</class>
 <widget class="QDialog" name="ZExceptionDLg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="MessageTEd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="ComplementTEd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ModuleLBl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>module</string>
   </property>
  </widget>
  <widget class="QLabel" name="ZStatusLBl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ZStatus</string>
   </property>
  </widget>
  <widget class="QLabel" name="SeverityLBl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Severity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBTn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKBTn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OtherBTn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Other</string>
   </property>
  </widget>
  <widget class="QLabel" name="LogoLBl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>logo</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Main message</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Complement</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Module</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Severity</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="dontShowCKb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Don't show again</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
